mkosi

```diff
@@ -3005,6 +3005,12 @@ def find_skeleton(args: CommandLineArguments) -> None:
     if os.path.isfile("mkosi.skeleton.tar"):
         args.skeleton_trees.append("mkosi.skeleton.tar")
 
+def args_find_path(args: CommandLineArguments, name: str, path: str) -> None:
+    if getattr(args, name) is not None:
+        return None
```

Comment:
why does this return `None` here? it shoud not return anything at all, no?

Reply:
Left-over from previous version of the code. I'll remove it.